src/components/shared/ColorSelect/ColorSelect.tsx

```diff
@@ -2,7 +2,9 @@ import React, { useCallback, useMemo, ReactNode, ComponentProps } from 'react';
 import { defineMessages } from 'react-intl';
 
 import { Appearance, Select, SelectOption } from '~shared/Fields';
-import ColorTag, { Color } from '~shared/ColorTag';
+import ColorTag from '~shared/ColorTag';
+
```

Comment:
is the empty line necessary?

Reply:
Can't remember why it's in here. I'll remove it.